nomos-blend/core/src/message_blend/mod.rs

```diff
@@ -98,8 +98,11 @@ where
                     tracing::error!("Failed to send message to the outbound channel: {e:?}");
                 }
             }
+            Err(e @ MessageUnwrapError::NotAllowed) => {
+                tracing::debug!("{e}");
```

Comment:
Should we start using log targets? This feels like `blend` or `blend::core` target to me?

Reply:
We are not using targets but spans. This logs would be under the blend span if it was set correctly in the service (I think it is).

Reply:
Actually, the target is added by unmodified tracing crate, `nomos-tracing` mostly contains open telemetry related layer creation helpers, which are used by `nomos-tracing-service`, and couple of macros to add `trace_id`. I think it would be nice to reexport `tracing` crate through `nomos-tracing`, that way there might be little less confusion.